console_bot/birthdays_per_week.py

```diff
@@ -21,6 +21,39 @@ def get_birthdays_per_week(users):
         return "No birthdays next week"
     return "\n".join([f"{date}: {(',').join(people)}" for date, people in result.items()])
 
+# Print users for 'users' list that have birthdays in the next number of days
+def get_birthdays_in_next_days(users, days):
+    current_date = datetime.today().date()
+    result = defaultdict(list)
+    for user in users:
+        user_name = user["name"]
+        user_birthday = user["birthday"].date()
+        birthday_this_year = prepare_birthday_date(current_date, user_birthday)
+        delta_days = (birthday_this_year - current_date).days
+        if delta_days < int(days):
+            result_dates(result, user_name, birthday_this_year)
+    if not result:
+        return f"No birthdays in next {days} days"
+    return "\n".join([f"{date}: {(',').join(people)}" for date, people in result.items()])
```

Comment:
```suggestion
    return "\n".join([f"{date}: {(",".join(people)}" for date, people in result.items()])
```
Python 3.10 doesn't accept the same type of quotes inside the quotes.